sqlmypy.py

```diff
@@ -141,14 +142,23 @@ def decl_info_hook(ctx):
 
         Base = declarative_base()
     """
+    cls_instances = []  # type: List[Instance]
+
+    if 'cls' in ctx.call.arg_names:
```

Comment:
This will work only for `declarative_base(cls=Base)`, but not for `declarative_base(Base)`. Is this deliberate?

Reply:
`declarative_base` has multiple arguments. the signature is `
declarative_base(bind=None, metadata=None, mapper=None, cls=object,
                     name='Base', constructor=_declarative_constructor,
                     class_registry=None,
                     metaclass=DeclarativeMeta)`

What is the standard way to get an argument's value in mypy ?

Reply:
At the semantic analysis stage it will be hard. I just wanted to understand the situation. Maybe just leave a comment that passing classes as positional arguments is not supported yet.